### PR TITLE
Hotfix: Add WorkerEmail to WarningNoteCaseNote

### PR DESCRIPTION
**Pull request by LBHBReynolds-Carr**
## Link to JIRA ticket

https://hackney.atlassian.net/browse/SCT-135

### *What is the problem we're trying to solve*

All worker email returns null from cases endpoint
Worker email not being created during casenote insertion

### *What changes have we introduced*

Worker email added to casenote insertion

#### _Checklist_

- [x] Added end-to-end (i.e. integration) tests where necessary e.g to test complete functionality of a newly added feature
- [x] Added tests to cover all new production code
- [x] Added comments to the README or updated relevant documentation _(add link to documentation)_, where necessary.
- [x] Checked all code for possible refactoring
- [x] Code pipeline builds correctly

### CODE REVIEW

File: SocialCareCaseViewerApi/V1/Gateways/DatabaseGateway.cs [hunk 1]
```diff
@@ -735,7 +735,8 @@ public PostWarningNoteResponse PostWarningNote(PostWarningNoteRequest request)
                 Note = $"{dt.ToShortDateString()} | Warning Note | Warning note created against this person",
                 FormNameOverall = "API_WarningNote",
                 FormName = "Warning Note Created",
-                WarningNoteId = warningNote.Id.ToString()
+                WarningNoteId = warningNote.Id.ToString(),
+                WorkerEmail = request.CreatedBy
```
Comment (Renny4Real) [inline SocialCareCaseViewerApi/V1/Gateways/DatabaseGateway.cs:739]: Do we have tests for this? Could we add a test?

**Review by Renny4Real — COMMENTED** [review]
Quick comment 🙏🏾

File: SocialCareCaseViewerApi/V1/Gateways/DatabaseGateway.cs (hunk 1 above)
Comment (LBHBReynolds-Carr) [reply SocialCareCaseViewerApi/V1/Gateways/DatabaseGateway.cs:739]: The test is in the `DatabaseGateWayTests` under `PostWarningNoteShouldCallInsertCaseNoteMethod`.

**Review by Renny4Real — APPROVED** [review]
:shipit:

File: SocialCareCaseViewerApi/V1/Gateways/DatabaseGateway.cs (hunk 1 above)
Comment (JohnFarrellDev) [reply SocialCareCaseViewerApi/V1/Gateways/DatabaseGateway.cs:739]: I'm inclined to think we do need a test or a change of an existing test here.

My thought is if we undid the code change you made would any test fail? Ideally our tests can help stop us introducing bugs or any regressions in the future.

Comment (Renny4Real) [reply SocialCareCaseViewerApi/V1/Gateways/DatabaseGateway.cs:739]: @JohnFarrellDev There was a brief convo about the testing with the MongoDB [here](https://hackit-lbh.slack.com/archives/C021Q88C953/p1620914341030000), but I agree that we should give testing this a try if we can.

**Pull request merged**